<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>461</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="InputLayout_1">
     <item>
      <widget class="QLabel" name="WaveLabel_1">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Wave 1</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="SineButton_1">
         <property name="text">
          <string>Generate Sine Wave</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="FreqDomain_1">
         <property name="text">
          <string>Frequency Domain</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="FrequencyBox_2">
       <item>
        <widget class="QSlider" name="FrequencySlider_1">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="FrequencyLabel_1">
         <property name="text">
          <string>600 Hz</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="TimeDomain" name="timeDomainInput_1" native="true"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>190</y>
      <width>461</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="InputLayout_2">
     <item>
      <widget class="QLabel" name="WaveLabel_2">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Wave 2</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="SineButton_2">
         <property name="text">
          <string>Generate Sine Wave</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="FrequencyBox_3">
       <item>
        <widget class="QSlider" name="FrequencySlider_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="FrequencyLabel_2">
         <property name="text">
          <string>600 Hz</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="TimeDomain" name="timeDomainInput_2" native="true"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>380</y>
      <width>461</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="outputLayout">
     <item>
      <widget class="QLabel" name="WaveLabel_3">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>AM Output</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="outputButton">
       <property name="text">
        <string>Generate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="TimeDomain" name="timeDomainOutput" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TimeDomain</class>
   <extends>QWidget</extends>
   <header>timedomain.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
